be/src/common/config.h

```diff
@@ -1810,4 +1810,7 @@ CONF_mInt64(max_batch_num_per_fetch_operator, "8");
 CONF_mInt64(max_chunk_num_per_fetch_batch, "8");
 CONF_mBool(enable_fetch_local_pass_through, "true");
 CONF_mInt64(max_lookup_batch_request, "8");
+// if the first predicate column's selectivity bigger than this, trigger sample,
+// which means if the selectivity is very good already, we don't need to sample
+CONF_mDouble(tigger_sample_selectivity, "0.2");
```

Comment:
typo.  trigger_sample_selectivity

this config name is bad. please give a clear and meaning name.